### DISCUSSION TITLE
Option to "save as" a quick note.

### DISCUSSION THREAD

**Discussion by twizzlebizzle** [gsantner/markor#1811] [Ideas] [answered]
I don't fully understand the quicknote function.  I use it currently when the app opens to start writing down some "quicknotes".  
  
When I'm done it just stays there, I either have to copy and paste the note into another markdown file if I want to keep it saved or just delete it.
  
It would be great if you could save as the quicknote into another file so it's saved/archived.

**Comment by gsantner** [comment]
What you do with information and how you organize it is up to how you like it.

You can select any Text and in the context menu where Copy/Cut is, there is also "Markor". There you have many possibilities to append/add/archive the text to where you want it in as well. This Markor button is also available in other apps so when you highlight some text in the web browser you can do that in the same way.

Otherwise, for todo.txt (the ToDo tab is one such) - there is archiving functionality to move off done todos/tasks to another file easily, like done.todo.txt.

**Comment by twizzlebizzle** [comment]
Sorry this was more of a feature suggestion to allow me to use Markor how I'd like it aha.  
  
I guess otherwise I need to copy the quicknote.md file?

**Reply by harshad1** [reply]
I keep a running list of notes / thoughs / memos in the quicknote file and I parse them out into the appropriate places (todo, personal wiki etc etc) every few days.

**Comment by harshad1** [comment, marked as answer]
Also note that you can share a note. This is effectively == save as.
